ci/.gitlab-ci.yml

```
@@ -45,7 +45,7 @@ setup_experiments:
     - ${HOMEGFS}/ci/scripts/utils/ci_utils_wrapper.sh create_experiment ${HOMEGFS}/ci/cases/pr/${caseName}.yaml
   parallel:
     matrix:
-      - caseName: ["C48_ATM", "C48mx500_3DVarAOWCDA", "C48mx500_hybAOWCDA", "C48_S2SWA_gefs", "C48_S2SW", "C96_atm3DVar", "C96C48_hybatmDA", "C96C48_hybatmaerosnowDA"]
+      - caseName: ["C48mx500_hybAOWCDA", "C96_atm3DVar", "C96C48_hybatmDA", "C96C48_ufs_hybatmDA", "C48_S2SW C96C48_hybatmaerosnowDA", "C48mx500_3DVarAOWCDA", "C48_ATM C48_S2SWA_gefs"]
```

Missing some quotes and commas:
```suggestion
      - caseName: ["C48mx500_hybAOWCDA", "C96_atm3DVar", "C96C48_hybatmDA", "C96C48_ufs_hybatmDA", "C48_S2SW", "C96C48_hybatmaerosnowDA", "C48mx500_3DVarAOWCDA", "C48_ATM", "C48_S2SWA_gefs"]